reference/speeduino.ini

```diff
@@ -380,7 +380,9 @@ page = 6
         boostKP      = scalar, U08,     57,             "%",        1.0,   0.0,  0.0,  200.0,      0 ; * (  1 byte)
         boostKI      = scalar, U08,     58,             "%",        1.0,   0.0,  0.0,  200.0,      0 ; * (  1 byte)
         boostKD      = scalar, U08,     59,             "%",        1.0,   0.0,  0.0,  200.0,      0 ; * (  1 byte)
-        unused6-60   = scalar, U08,      60,        "RPM",       100.0,       0.0,   100,     25500,    0
+	
+	lnchPullRes  =   bits, U08,      60         [0:1],      "Float"  , "Pullup", "INVALID", "INVALID"
+        unused6-60   =   bits, U08,      60,        [2:7],      "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID", "INVALID"

```

Comment:
Super small detail, but you can't have a bits entry with all INVALID options. Just make the first one a value of 'ONE' or something. TS chucks an error message when loading the project otherwise.